### PR TITLE
Add a conditional check if key in available_feeds: before accessing available_feeds[key].

### PR DESCRIPTION
**Pull request by tomhuhh**
Added a conditional check if key in available_feeds: before accessing available_feeds[key]. This is to address issue #1135.

### CODE REVIEW

File: RUFAS/routines/animal/ration/ration_driver.py [hunk 1]
```diff
@@ -368,28 +368,29 @@ def report_ration(cls, ration: Dict, available_feeds: Dict) -> Tuple[Dict, Dict]
...
-                        denom = 6.38
-                    # [A.2.A.1]
-                    else:
-                        denom = 6.25
-                    nutrient_amount[nutr] += (available_feeds[key]["CP"] / (denom * 100)) * val
-                else:
-                    if nutr == 'DE':
-                        if available_feeds[key]['DE'] != -1:
-                            nutrient_amount[nutr] += val * (available_feeds[key]['DE'] / 100)
+            if key in available_feeds:
```
Comment (ew3361zh) [inline RUFAS/routines/animal/ration/ration_driver.py:371]: What if the key is not in the available_feeds? Do you just want it to be ignored silently? Or would it be helpful to add an else statement which adds a warning to OM noting the key doesn't exist?

File: RUFAS/routines/animal/ration/ration_driver.py [hunk 2]
```diff
@@ -368,28 +368,29 @@ def report_ration(cls, ration: Dict, available_feeds: Dict) -> Tuple[Dict, Dict]
 
         for key, val in ration.items():
             nutrient_amount["dm"] += val
```
Comment (ew3361zh) [inline RUFAS/routines/animal/ration/ration_driver.py:370]: Do you still want the value for the key added to the nutrient amount's 'dm' if the key is invalid?

**Review by ew3361zh — COMMENTED** [review]
A couple quick questions for you to consider - please just ask for my re-review after they've been addressed.

**Comment by tomhuhh** [comment]
Closing this PR because it is not applicable anymore... Will check and create (if it does not exist) an issue regarding the fact that the calf ration is hard coded.

**Pull request closed without merging**